src/model/Game.ts

```diff
@@ -19,11 +20,11 @@ export class Game{
     gameMap:GameMap;
 
     constructor(host: Player, name: string){
-        this.playersJoined= [];
+        this.playersJoined= [new Player('Betty the Bot', PlayerColor.Red)];
```

Comment:
Just one question. Should we throw in something to make sure no other players are red?

Reply:
Good call. I didn't realize that colors aren't assigned until later, so I fixed Betty's initial color to None.